src/IB/IBFEMethod.cpp

```diff
@@ -1758,22 +2262,36 @@ IBFEMethod::computeStressNormalization(PetscVector<double>& Phi_vec,
     // Setup extra data needed to compute stresses/forces.
 
     // Extract the FE systems and DOF maps, and setup the FE objects.
-    LinearImplicitSystem& Phi_system = equation_systems->get_system<LinearImplicitSystem>(PHI_SYSTEM_NAME);
+    TransientLinearImplicitSystem& Phi_system =
+        equation_systems->get_system<TransientLinearImplicitSystem>(PHI_SYSTEM_NAME);
     const DofMap& Phi_dof_map = Phi_system.get_dof_map();
     FEDataManager::SystemDofMapCache& Phi_dof_map_cache = *d_fe_data_managers[part]->getDofMapCache(PHI_SYSTEM_NAME);
     std::vector<unsigned int> Phi_dof_indices;
     FEType Phi_fe_type = Phi_dof_map.variable_type(0);
     std::vector<int> Phi_vars(1, 0);
 
+    // things for building RHS of Phi linear system based on poisson solver.
```

Comment:
These parameter names _might_ be a little clearer in the input files if they were grouped together in their own `Database`, so that you don't have to worry about prefixing them with `cg` or `ipdg`

In the input file, you could have
```
IBFEMethod {
   // other settings...
   StressSplitting {
      penalty = ...
      formulation = ...
   }
}
```
You can access this sub-`Database` via
```
    if (input_db->isDatabase("StressSplitting"))
    {
        Pointer<Database> stress_splitting_db = input_db->getDatabase("StressSplitting");
        // ... etc ...
    }
```

Reply:
yes, this would be much cleaner!!